shared/middleware/helpers/storage_apps.rb

```diff
@@ -13,10 +13,12 @@ def initialize(storage_id)
   end
 
   def create(value, ip_address)
+    timestamp = Time.now
```

Comment:
If it's not too annoying, let's switch this to DateTime instead of Time. It looks like we're already using DateTime everywhere else in shared/middleware, and this is one of very few places we use Time.